src/lib/config.js

```diff
@@ -59,9 +59,7 @@ function getInitialConfig () {
 
 var initialConfig = getInitialConfig()
 
-if (isDefinedConfigValue(process.env.SERVER_NAME)) {
-  initialConfig.serverName = process.env.SERVER_NAME
-}
+initialConfig.serverName = process.env.SERVER_NAME || 'Composr 2.0'
```

Comment:
Initially server name should be loaded from config.json. it should never be undefined if your config.json has the value setted.